app/helpers.php

```diff
@@ -414,6 +414,73 @@ function get_valid_locale($requestedLocale)
     }
 }
 
+function hsl_to_hex($h, $s, $l)
+{
+    $r = $l;
+    $g = $l;
+    $b = $l;
+
+    $v = $l <= 0.5 ? $l * (1.0 + $s) : $l + $s - $l * $s;
```

Comment:
the input saturation and lightness are in different unit. the one used here makes more sense though so just need to fix the value being passed